Update `mock_call` docs (#980)
software-mansion/protostar: website/docs/tutorials/07-testing/02-cheatcodes/mock-call.md

def mock_call(contract_address: int, fn_name: str, ret_data: Union[List[int], Dict]) -> Callable: ...
```

Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address`, until the returned callable is called. 

Mocked call returns data provided in `ret_data`.

> Mock works globally, for all contracts, not only the testing contract.

Reviewer: I mean this PR doesn't change anything rn. Cannot we just close it? (I am ok if we want split it into separate lines)

def mock_call(contract_address: int, fn_name: str, ret_data: Union[List[int], Dict]) -> Callable: ...
```

Mocks all calls to a `@view` function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. 
Mocked call returns data provided in `ret_data`. 

## Caveats

> - The cheatcode works only for contract's exposed `@view`s. Doesn't work for internal functions, `@external`'s etc.

Reviewer 2: ```suggestion
- This cheatcode works only for contract's exposed `@view`s. It does not work for internal functions, `@external`s etc.
```

> - This cheatcode works only for contract's exposed `@view`s. It does not work for internal functions, `@external`s etc.

Reviewer: I dont think it is a correct information. Please check it.

def mock_call(contract_address: int, fn_name: str, ret_data: Union[List[int], Dict]) -> Callable: ...
```

> Mocks all calls to a `@view` function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. 

Reviewer: This is still incorrect

Author: damn, you're right
